layers/+tags/gtags/packages.el

```diff
@@ -11,13 +12,24 @@
 
 (setq gtags-packages
   '(
+    eldoc
     helm-gtags
     ggtags
     ))
 
 (defun gtags/init-ggtags ()
   (use-package ggtags
-    :defer t))
+    :defer t
+    :init
+    (progn
+      ;; modes that do not have a layer, add here.
+      (add-hook 'awk-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'shell-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'tcl-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
```

Comment:
Bar <notifications@github.com> writes:
>> +      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
>
> It should be `(add-hook <mode-hook> #'ggtags-mode)`.
> Calling `ggtags-mode` without an argument turns it on (when called from elisp code, like in this case).

I didn't know this! I will make the change, thanks!

Chris.